Triangulation_2/doc/Triangulation_2/CGAL/Constrained_triangulation_plus_2.h

```diff
@@ -293,6 +293,21 @@ std::size_t insert_constraints(PointIterator points_first, PointIterator points_
                                IndicesIterator indices_first, IndicesIterator indices_last);
 
 
+/*! 
+splits into polylines the graph `g` at vertices of degree greater than 2
+and at vertices for which `is_terminal(v)==true`.
+
+\warning all existing constraints will be discarded.
+
+\param is_terminal An optional function returning true if the vertex
+`v` of degree 2 is a polyline endpoint and false otherwise. If no
+function is provided, a default function is provided that returns true
+for vertices whose degree is different from 2.
+
+\sa `split_graph_into_polylines()`
+*/ 
+void split_subconstraint_graph_into_constraints(const std::function<bool(Vertex_handle)>& is_terminal);
```

Comment:
This documentation refers to a graph `g` that is not introduced. And that is not clear either what is the exact behavior of the function.

Reply:
Indeed. I fixed it.